README.md

```diff
@@ -13,7 +13,9 @@ The apps that you will be creating in Mod 2 will make heavy use of these four ac
 
 Let's use Rails to build an application where we can manage some tasks.
 
-We're going to follow the MVC design pattern, which Rails uses by default, to implement the CRUD actions for our Task Manager app. For a quick explanation of MVC, please read through [this](https://medium.freecodecamp.org/simplified-explanation-to-mvc-5d307796df30) post.
+TODO: Add an image very similar to this but without the view portion: https://backend.turing.edu/module2/lessons/images/mvc_rails.png
+
+TODO: Update MVC wording for an API returning JSON data. Maybe use a page like this to learn more about JSON? https://backend.turing.edu/module3/notes/json_for_api_development.html
```

Comment:
I wonder, since we'll introduce JSON in more detail in the How the Web Works Class, could we just say here something like:

In this tutorial we're building an API. APIs are used to provide data to an external system when they request it, and the the APIs we'll see will mostly serve up data in the form of JSON. JSON is a popular data format that looks like a Ruby hash. 

just to keep things simple?

Reply:
I like this. Those who are super curious always end up googling it on their own anyways. They will explore further if they wish, otherwise they'll get it during the mod.

Reply:
Good call Juliet, I like this plan!